docs/realtime/changes/index.de.md

```diff
@@ -0,0 +1,104 @@
+# GTFS Realtime Änderungen
+
+Die GTFS-Realtime Spezifikation ist nicht in Stein gemeißelt. Vielmehr handelt es sich um eine offene Spezifikation, die von der Gemeinschaft der Verkehrsunternehmen, Entwickler und anderen Beteiligten, die GTFS Realtime nutzen, entwickelt und gepflegt wird. Es wird erwartet, dass diese Gemeinschaft von Produzenten und Konsumenten von GTFS Realtime Daten Vorschläge zur Erweiterung der Spezifikation machen wird, um neue Funktionen zu ermöglichen.
+
+Wenn Sie einen Beitrag zu GTFS Realtime leisten möchten, lesen Sie den [Prozess zur Änderung der Spezifikation](../process) und verfolgen Sie die Diskussionen in den offenen [Fragen](https://github.com/google/transit/issues) und [Pull-Requests](https://github.com/google/transit/pulls) im Transit-Github-Repository von Google[(google/transit](https://github.com/google/transit)). ![](../../assets/mark-github.svg)
+
+!!! note ""
+
+    Die offizielle Spezifikation, Referenz und Dokumentation sind in englischer Sprache verfasst. Weicht eine Translation in eine andere Sprache vom englischen Original ab, hat letztere Vorrang. Die gesamte Kommunikation erfolgt in englischer Sprache.
+
+<!-- <br><div class="landing-page">
+    <a class="button" href="../process">Prozess zur Änderung der Spezifikation</a><a class="button" href="../guiding-principles">Leitprinzipien</a><a class="button" href="../revision-history">Historie der Revision</a><a class="button" href="../extensions">Echtzeit-Erweiterungen</a>
+</div> -->
+
+## Aktive Proposals![](../../assets/pr-active.svg)
+
+<!-- Aktive Vorschläge für neue Funktionen in GTFS Realtime.  -->
+
+Beteiligen Sie sich an den Diskussionen auf [Github](https://github.com/google/transit/pulls)!
+
+<!-- <div class="row">
```

Comment:
What's the purpose of these comments? To they cause something to be rendered?

Reply:
They're previous drafts for how we could display content.